src/inc/crsttypes.h

```diff
@@ -189,7 +189,8 @@ enum CrstType
     CrstVSDIndirectionCellLock = 170,
     CrstWinRTFactoryCache = 171,
```

Comment:
This file should be auto-generated from `crsttypes.def`. I do not see any changes in `crsttypes.def`.

Could you please update `crsttypes.def` and run the auto-generator?

Reply:
Interesting, I didn't know it was auto generated.

Reply:
Yes, I think the root problem here is that the auto-generator inserts the tabs. Some mild tweaking of the autogenerator to produce spaces instead is probably in order. In fact, I have such a change sitting in one of my branches... give me a moment and I'll put it together as an independent PR.

Reply:
See https://github.com/dotnet/coreclr/pull/21029